css/types/transform-function.json

```diff
@@ -192,6 +192,44 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "none": {
+            "__compat": {
+              "description": "`perspective(none)`",
+              "spec_url": "https://drafts.csswg.org/css-transforms-2/#funcdef-perspective",
+              "tags": [
+                "web-features:transforms3d"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": "97"
```

Comment:
✅ https://github.com/chromium/chromium/commit/ae6f9bdb7e14b19650ee21cba03418ed35f1a2f5 in [97.0.4672.0](https://github.com/chromium/chromium/releases/tag/97.0.4672.0) before 97.0.4692 added this.

Bug: https://issues.chromium.org/issues/40180358#comment13